platforms/documentation/docs/src/docs/userguide/dep-man/dependency-management/dependency_locking.adoc

```diff
@@ -0,0 +1,301 @@
+* `*:guava`: will let all modules named `guava`, whatever their group, update
+* `org.springframework.spring*:spring*`: will let all modules having their group starting with `org.springframework.spring` and name starting with `spring` update
+
+NOTE: The resolution may cause other module versions to update, as dictated by the Gradle resolution rules.
+
+== Disabling dependency locking
+
+To disable dependency locking for a configuration:
+
+1. **Remove Locking Configuration:**
+Ensure that the configuration you no longer want to lock is not configured with dependency locking. This means removing or commenting out any `activateDependencyLocking()` calls for that configuration.
+
+2. **Update Lock State:**
+The next time you update and save the lock state (using the `--write-locks` option), Gradle will automatically clean up any stale lock state associated with the configurations that are no longer locked.
+
+NOTE: Gradle must resolve a configuration that is no longer marked as locked to detect and drop the associated lock state. Without resolving the configuration, Gradle cannot identify which lock state should be cleaned up.
+
+[[ignoring_dependencies]]
+== Ignoring specific dependencies from the lock state
+
+In some scenarios, dependency locking may not be focused on reproducibility.
+
+As a build author, you might want certain dependencies to update more frequently than others.
+For example, internal dependencies within an organization might always use the latest version, while third-party dependencies follow a different update cycle.
+
+CAUTION: This approach can compromise reproducibility. Consider using <<#fine_tuning_dependency_locking_behaviour_with_lock_mode,different lock modes>> or <<#configuring_the_per_project_lock_file_name_and_location,separate lock files>> for specific cases.
+
+You can configure dependencies to be ignored in the `dependencyLocking` project extension:
+
+====
+include::sample[dir="snippets/dependencyManagement/dependencyLocking-lockingSingleFilePerProject/kotlin",files="build.gradle.kts[tags=ignore-dependencies]"]
+include::sample[dir="snippets/dependencyManagement/dependencyLocking-lockingSingleFilePerProject/groovy",files="build.gradle[tags=ignore-dependencies]"]
+====
+
+The notation `<group>:<name>` is used to specify dependencies, where `\*` acts as a trailing wildcard. Note that `*:*` is not accepted, as it effectively disables locking.
+See <<#selectively_updating_lock_state_entries,the description>> on updating lock files for more details.
+
+Ignoring dependencies will have the following effects:
+
+- Ignored dependencies apply across all locked configurations, and the setting is project-wide.
+- Ignoring a dependency does not exclude its transitive dependencies from the lock state.
+- No validation ensures that an ignored dependency is present in any configuration resolution.
+- If the dependency is present in the lock state, it will be filtered out when loading.
```

Comment:
💅 When loading what?

Reply:
I asked the team but was not able to get an answer. I left it as the old wording. My understanding was loading the lock state.